docs/README.md

```diff
@@ -1,6 +1,6 @@
 # Wagtail docs
 
-These are Sphinx docs, automatically built at [https://docs.wagtail.org](https://docs.wagtail.org) when the `main` branch is committed to Github. To build them locally, install Wagtail's development requirements (in the root Wagtail directory):
+These are Sphinx docs, automatically built into [Wagtail's Documentation](https://docs.wagtail.org) when the `main` branch is committed to Github. To build them locally, install Wagtail's development requirements (in the root Wagtail directory):
```

Comment:
I think the URL used here is fine as is, this is for developers who are working on the documentation and unless there is a critical confusion with this URL I think it should probably stay as is. 

Note that the developer reading this already has some understanding of the documentation as they have had to read it to learn how to review the documentation. The title on this page also says Wagtail docs.

Reply:
Ok, thanks for the view. I just thought that it be better to also build for non-technical people as well. Coming to think of it only technical people have access to this documentation